config/initializers/git_sha.rb

```diff
@@ -19,7 +19,7 @@
 LAST_DEPLOYED =
   if Rails.env.production? && File.exist?('/opt/epigaea/revisions.log')
     deployed = `tail -1 /opt/epigaea/revisions.log`.chomp.split(" ")[7]
-    Date.parse(deployed).strftime("%d %B %Y")
+    DateTime.parse(deployed).strftime("%e %b %Y %H:%M:%S") # rubocop:disable Rails/TimeZone
```

Comment:
Can you put the rubocop exception into the `.rubocop.yml` file instead of in the source code.  I feel like sprinkling rubocop rules all over the code will needlessly clutter up the code.